Fabric.Identity.API/scripts/Install-IdPSS-Utilities.psm1

```diff
@@ -314,6 +349,16 @@ function Confirm-Tenants {
     }
 }
 
+function Get-InstallIdPSSUtilsUserConfirmation {
+    Write-DosMessage -Level "Information" -Message "Found ${$filterKeys.count} duplicate secrets."
+    $deleteSecrets = Read-Host  "Delete duplicate secret(s)? [Y/N]"
+    switch ($deleteSecrets) {
+        Y {return $true}
```

Comment:
is this taking into account lower case Y as well?

Reply:
Yes, this defaults to a case insentive check